deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
```

Comment:
How is this file getting generated? Entirely written by hand? It looks too complex for that, but there's also no make rule for it.

Reply:
That didn't work for me:
```
$ rm deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml 
$ make operator-generate-csv
GOROOT=/nvme/gopath/go-1.14.4 _work/bin/operator-sdk-v0.18.2 generate crds --crd-version v1beta1
INFO[0000] Running CRD generator.                       
INFO[0000] CRD generation complete.                     
cd deploy/operator && cp ../common/pmem-csi.intel.com*_cr.yaml ./
GOROOT=/nvme/gopath/go-1.14.4 WATCH_NAMESAPCE=default _work/bin/operator-sdk-v0.18.2 generate csv --make-manifests=true \
        --operator-name pmem-csi-operator --interactive=false --update-crds=true\
        --apis-dir=/pkg/apis --crd-dir=./deploy/crds --deploy-dir=./deploy/operator \
        --csv-version 0.7.0
INFO[0000] Generating CSV manifest version 0.7.0        
FATA[0000] error generating CSV: error reading existing ClusterServiceVersion base deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml: open deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml: no such file or directory 
make: *** [operator/operator.make:25: operator-generate-csv] Error 1
```

Reply:
Is this working now?

Reply:
Yes, now it supposed to build the catalog from zero using `make operator-generate-catalog`.

Reply:
Still didn't work for me:
```
$ make operator-generate-catalog VERSION="0.7.0"
mkdir -p _work/bin/ 2> /dev/null
curl -L https://github.com/operator-framework/operator-sdk/releases/download/v1.0.0/operator-sdk-v1.0.0-x86_64-linux-gnu -o /nvme/gopath/src/github.com/intel/pmem-csi/_work/bin/operator-sdk-v1.0.0
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   649  100   649    0     0   2627      0 --:--:-- --:--:-- --:--:--  2627
100 66.6M  100 66.6M    0     0  17.9M      0  0:00:03  0:00:03 --:--:-- 21.0M
chmod a+x /nvme/gopath/src/github.com/intel/pmem-csi/_work/bin/operator-sdk-v1.0.0
Generating manifests ...
_work/bin/operator-sdk-v1.0.0 generate kustomize manifests --apis-dir ./pkg/apis/ --interactive=false --output-dir=deploy/manifests && \
make operator-generate-crd && \
mkdir -p deploy/manifests/default deploy/manifests/samples && \
echo -e "bases: [../crd, ../../kustomize/operator]\nimages:\n- name: intel/pmem-csi-driver\n  newTag: 0.7.0" > deploy/manifests/default/kustomization.yaml && \
echo "resources: [../../common/pmem-csi.intel.com_v1alpha1_deployment_cr.yaml]" > deploy/manifests/samples/kustomization.yaml && \
sed -i -e 's;- ../scorecard;;g' -e 's;../;;g' deploy/manifests/kustomization.yaml
Generating kustomize files in deploy/manifests
Kustomize files generated successfully
make[1]: Entering directory '/nvme/gopath/src/github.com/intel/pmem-csi'
bash: -c: line 2: syntax error: unexpected end of file
make[1]: *** [operator/operator.make:21: controller-gen] Error 1
make[1]: Leaving directory '/nvme/gopath/src/github.com/intel/pmem-csi'
make: *** [operator/operator.make:48: operator-generate-manifests] Error 2
```

It's also not obvious what VERSION means in this context and how to use it when generating the catalog. Can you write documentation for this in DEVELOPMENT.md?

Reply:
> Still didn't work for me:

My bad, The script that deals with installing the `controller-gen` were broken. This was not an issue at my end as the binary available on my machine. Now I fixed this. Hopefully, it should work now.

> It's also not obvious what VERSION means in this context and how to use it when generating the catalog. Can you write documentation for this in DEVELOPMENT.md?

Extended the `Release management` section to list the steps to follow to release the operator to OperatorHub.